client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/AbstractRemoteShuffleInputGateFactory.java

```diff
@@ -67,38 +64,30 @@ public abstract class AbstractRemoteShuffleInputGateFactory {
   protected CelebornConf celebornConf;
 
   public AbstractRemoteShuffleInputGateFactory(
-      Configuration flinkConf,
-      CelebornConf conf,
-      NetworkBufferPool networkBufferPool,
-      int networkBufferSize) {
+      CelebornConf conf, NetworkBufferPool networkBufferPool, int networkBufferSize) {
     this.celebornConf = conf;
-    long configuredMemorySize =
-        FlinkUtils.byteStringValueAsBytes(flinkConf, PluginConf.MEMORY_PER_INPUT_GATE);
-    long minConfiguredMemorySize =
-        FlinkUtils.byteStringValueAsBytes(flinkConf, PluginConf.MIN_MEMORY_PER_GATE);
+    long configuredMemorySize = celebornConf.memoryPerInputGate();
```

Comment:
if we copy the configs from flinkConf to celebornConf, the flinkConf can be removed directly, otherwise the config value is setted in flinkConf not in celebornConf, so we can't get the real value setted by user.